agent/trend_watcher.py

```
@@ -641,6 +830,53 @@ def _dispatch_to_improvement_loop(self, alerts: list[TrendAlert]) -> None:
+                    "context": {
+                        "source": "trend_watcher",
+                        "relevance_score": alert.relevance_score,
+                        "dispatched_at": time.time(),
+                    },
+                }
+                asyncio.run(_httpx.AsyncClient().post(                        f"{hermes_url.rstrip('/')}/tasks",
+                    json=payload,
+                    timeout=10,
+                ))
+                log.info("TrendWatcher: dispatched alert %s to Hermes", alert.title[:50])
+            except Exception as exc:                    log.warning("TrendWatcher: Hermes dispatch failed for '%s': %s", alert.title[:40], exc)
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)